.github/workflows/cspell.yml

```diff
@@ -0,0 +1,25 @@
+name: cspell
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 2
+  NODE: 16
+
+jobs:
+  cspell:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run cspell
+        uses: streetsidesoftware/cspell-action@v1.3.5
+        with:
+          files: "site/**/*.md"
```

Comment:
I'm wondering if we can move this in .cspell.json too? That way everything should work without any arguments.

Reply:
Don't know if it can help but instead of `cspell \"site/**/*.md\" --no-progress`, it it possible to manually launch `cspell --no-progress` and to define the path to analyze in your `.cspell.json` by adding:
```json
"files": [
  "site/**/*.md"
]
```
It will maybe work as well with `streetsidesoftware/cspell-action@v1.3.5`.

Reply:
Perfect, I'll try that, thanks!

@coliff can you confirm if the above works for your use case through an editor?

BTW there's a `gitignore` option we could add in the config. I'm trying to figure out if it's enabled by default or not.